libs/npm/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "fine-uploader": "5.14.5"
```

Comment:
I agree. Only thing I found a little shaky is, that maintainer currently [wants to be replaced](https://github.com/FineUploader/fine-uploader/issues/1881).